SentryIntegration.swift

```diff
@@ -0,0 +1,124 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Sentry
+
+public class SentryIntegration {
+    public static let shared = SentryIntegration()
+    
+    public static var crashedLastLaunch: Bool {
+        return Client.shared?.crashedLastLaunch() ?? false
+    }
+    
+    private let SentryDSNKey = "SentryDSN"
+    private let SentryDeviceAppHashKey = "SentryDeviceAppHash"
+    private let DefaultDeviceAppHash = "0000000000000000000000000000000000000000"
+    private let DeviceAppHashLength = UInt(20)
+    
+    private var enabled = false
+    
+    private var attributes: [String : Any] = [:]
+    
+    public func setup(sendUsageData: Bool) {
+        assert(!enabled, "SentryIntegration.setup() should only be called once")
+        
+        if AppInfo.isSimulator() || !sendUsageData {
+            return
+        }
+        
+        guard let dsn = Bundle.main.object(forInfoDictionaryKey: SentryDSNKey) as? String, !dsn.isEmpty else {
+            return
```

Comment:
Also print to the console here to indicate that the DSN was unable to be obtained.